modules/ROOT/pages/database-administration/index.adoc

```diff
@@ -1,114 +1,80 @@
 
 [NOTE]
 ====
 Be aware that the automatically created _initial_ default database may have a different topology to the default configuration values.
 See xref:clustering/clustering-advanced/default-database.adoc[Default database in a cluster] for more information.
 ====
 
-Composite database::
-A Composite database is a logical grouping of multiple graphs contained in other, standard databases.
-+
-A Composite database defines an _execution context_ and a (limited) _transaction domain_.
-+
-For more information, see xref:composite-databases/index.adoc[Composite databases].
-
-The following image illustrates a default installation, including the `system` database and a single database named `neo4j` for user data:
-
-image::manage-dbs-community.png[title="A default Neo4j installation.", role="middle"]
-
-.Editions
-[NOTE]
-====
-The edition of Neo4j determines the number of possible databases:
+[role=enterprise-edition]
+[[manage-databases-home]]
+=== Per-user home databases
 
-* Installations of Community Edition can have exactly *one* user database.
-* Installations of Enterprise Edition can have any number of user databases.
+The home database is the database that you connect to by default when no database is specified.
+It is different from the default database, which is the database that the server uses when no home database is specified.
 
-All installations include the `system` database.
-====
+Per-user home databases are controlled via the Cypher administration commands.
 
+To set a home database for a user, this user must exist as a record in Neo4j.
+Therefore, for deployments using xref:authentication-authorization/introduction.adoc[auth providers] other than native, you create a native user with a matching username and then set a home database for that user.
+For more information on creating native users and configuring a home database for a user, see link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/access-control/manage-users[Cypher Manual -> User Management].
 
 [[manage-databases-system]]
 == The `system` database
 
-All installations include a built-in database named `system`, which contains meta-data and security configuration.
+All installations include a built-in database named `system`, which contains metadata on the DBMS and security configuration.
 
 The `system` database behaves differently than all other databases.
-In particular, when connected to this database you can only perform a specific set of administrative functions, as described in detail in link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/administration/databases[Cypher Manual -> Database management].
+In particular, when connected to this database you can only perform a specific set of administrative functions, as described in detail in  xref::database-administration/standard-databases/manage-databases.adoc[Database management].
```

Comment:
you can do both database, alias, server, access control commands on system -> should this sentence be extended to not just say database management?

Reply:
I don't think we need a link here, actually.
Something like:
'In particular, when connected to this database you can only perform a specific set of administrative functions, such managing databases, aliases, servers, and access control.'
will suffice.

Reply:
I'm not sure about calling them functions? we don't want to risk confusion around cypher functions like `abs` or `size`

Reply:
I went with 'tasks' instead.